tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1738,7 +1738,8 @@ def runTest(self):
 
                     print("pkts sent: %d, lower bound: %d, actual headroom pool watermark: %d, upper_bound: %d" % (
                         wm_pkt_num, expected_wm, hdrm_pool_wm, upper_bound_wm), file=sys.stderr)
-                    assert(expected_wm <= hdrm_pool_wm)
+                    if 'innovium' not in self.asic_type:
```

Comment:
If this change is only for Innovium platform, should the logic be something like: `if 'innovium' in self.asic_type, then do something specific for innovium platform`?

Reply:
We wanted to skip this assertion on Innovium platform, hence I added this one line to proceed only if it is NOT Innovium ASIC.